frontend/javascripts/oxalis/model/bucket_data_handling/mappings.js

```diff
@@ -10,46 +10,42 @@ import type { APIMapping } from "types/api_flow_types";
 import type { ProgressCallback } from "libs/progress_callback";
 import { createUpdatableTexture } from "oxalis/geometries/materials/plane_material_factory_helpers";
 import { doWithToken } from "admin/admin_rest_api";
-import { getMappings, getLayerByName } from "oxalis/model/accessors/dataset_accessor";
+import {
+  getMappings,
+  getMappingInfo,
+  getLayerByName,
+} from "oxalis/model/accessors/dataset_accessor";
 import { getRenderer } from "oxalis/controller/renderer";
 import { listenToStoreProperty } from "oxalis/model/helpers/listener_helpers";
 import { setMappingAction, setMappingEnabledAction } from "oxalis/model/actions/settings_actions";
-import type DataLayer from "oxalis/model/data_layer";
 import ErrorHandling from "libs/error_handling";
 import Request from "libs/request";
 import Store, { type Mapping, type MappingType } from "oxalis/store";
 import UpdatableTexture from "libs/UpdatableTexture";
 import messages from "messages";
-import { trackAction } from "oxalis/model/helpers/analytics";
 
 export const MAPPING_TEXTURE_WIDTH = 4096;
 export const MAPPING_COLOR_TEXTURE_WIDTH = 16;
 
 type APIMappings = { [string]: APIMapping };
 
-// For now, since we have no UI for this
-export function setupGlobalMappingsObject(segmentationLayer: DataLayer) {
+// Remove soon (e.g., October 2021)
```

Comment:
Out of curiosity, did you have a look whether these functions were called?

Reply:
I just had a look and it seems to have to not been tracked at least. The overall event count was quite low. I suppose, lots of users have an adblocker :thinking: